Fabric.Authorization.Domain/Services/GroupService.cs

```diff
@@ -65,9 +66,13 @@ public async Task<Group> UpdateGroup(Group group)
 
         public async Task<Group> GetGroup(GroupIdentifier groupIdentifier, string clientId)
         {
-            var group = await _groupStore.Get(groupIdentifier);             
-            var clientRoles = (await _roleService.GetRoles(clientId)).ToList();
-            group.Roles = clientRoles.Intersect(group.Roles).ToList();
+            var group = await _groupStore.Get(groupIdentifier);
+            if (!string.Equals(clientId, AccessControl.ClientId, StringComparison.OrdinalIgnoreCase))
```

Comment:
We are saying that if we are looking at any other group besides "Fabric-Access-Control", take the groups roles and only return the ones that are based on the Client's roles and the Groups roles.  

So the difference was that we were doing this, even for "Fabric-Access-Control".  Now when you get the group for "Fabric-Access-Control", it will have all roles.  Is that effectively saying that any role associated outside "Fabric-Access-Control" can edit it?

Reply:
No. This is just allowing the `fabric-access-control` client to retrieve roles that are not "owned" by it (see the `ClientOwner` field on the `SecurableItems` table. Users still need to have certain permissions to edit groups tied to a particular grain/securable item.